### PR TITLE
Change typo function name initOrderStatutsList

### PR DESCRIPTION
**Pull request by taoufiqaitali**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  |name of functions is incorrect, changed from initOrderStatutsList to initOrderStatusList, should be in english.
| Type?         | refacto 
| Category?     | BO
| BC breaks?    | no
| Deprecations? |  no
| Fixed ticket? | 
| How to test?  | name of function in file is incorrect.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21360)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

**Review by Sinepel — DISMISSED** [review]

**Review by atomiix — COMMENTED** [review]
Couldn't it be considered as a BC Break?

**Comment by Sinepel** [comment]
> Couldn't it be considered as a BC Break?

I don't know if my voice counts but I think it does :)

This function can be overridden

**Comment by matks** [comment]
> I don't know if my voice counts but I think it does :)

Yes it does 😄 

You know, there is also very few differences between maintainers and contributors
- maintainers can review PRs and contributors can do it too
- maintainers can discuss issues (bug reports and feature requests) and contributors can do it too
- maintainers can submit PRs and contributors can do it too
- maintainers can discuss with users (answering questions, discussing ideas) and contributors can do it too

The only thing maintainers can do that you cant is approve pull requests which unlock the merge button 😉 and that is just a tiny part.

**Review by matthieu-rolland — COMMENTED** [review]
I also think that this is a BC break, if an existing module extends this class and calls this method then it will break, or if it overrides this method and use `parent::initOrderStatutsList()`

**Review by Progi1984 — DISMISSED** [review]

File: controllers/admin/AdminStatusesController.php [hunk 1]
```diff
@@ -64,7 +64,7 @@ public function init()
     /**
      * init all variables to render the order status list.
      */
-    protected function initOrderStatutsList()
```
Comment (Progi1984) [inline controllers/admin/AdminStatusesController.php:67]: @taoufiqaitali Hi, could you create an another protected method `initOrderStatutsList` which call `initOrderStatusList` for removing the BC Break ? Thanks

**Review by Progi1984 — COMMENTED** [review]
Added a comment for removing the BC Break

File: controllers/admin/AdminStatusesController.php (hunk 1 above)
Comment (taoufiqaitali) [reply controllers/admin/AdminStatusesController.php:67]: @Progi1984 ok done, i created function that call initOrderStatusList

**Review by Progi1984 — DISMISSED** [review]

File: controllers/admin/AdminStatusesController.php [hunk 2]
```diff
@@ -123,6 +123,16 @@ protected function initOrderStatutsList()
...
     }
+    
+    /**
+     * init all variables to render the order status list.
+     *
+     * @deprecated renamed this to initOrderStatusList
+     */
+    protected function initOrderStatutsList()
+    {
+        return $this->initOrderStatusList();
```
Comment (Progi1984) [inline controllers/admin/AdminStatusesController.php]:
```suggestion

    /**
     * Init all variables to render the order status list.
     *
     * @deprecated Use `initOrderStatusList`
     */
    protected function initOrderStatutsList()
    {
        $this->initOrderStatusList();
```

File: controllers/admin/AdminStatusesController.php [hunk 3]
```diff
@@ -64,7 +64,7 @@ public function init()
     /**
      * init all variables to render the order status list.
      */
-    protected function initOrderStatutsList()
+    protected function initOrderStatusList()
```
Comment (Progi1984) [inline controllers/admin/AdminStatusesController.php]:
```suggestion
    protected function initOrderStatusList(): void
```

**Review by Progi1984 — COMMENTED** [review]
Small improvements. And it's OK.

**Comment by taoufiqaitali** [comment]
> improvements

**Comment by taoufiqaitali** [comment]
> Small improvements. And it's OK.

improvements done, thanks

**Review by Progi1984 — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by atomiix** [comment]
@PrestaShop/prestashop-core-developers Do we need QA on this one?

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
No need QA.
Thank you @taoufiqaitali

**Pull request merged**